### PR TITLE
Remove ?y test

### PR DESCRIPTION
**Pull request by Obaied**

### CODE REVIEW

File: aaa [hunk 1]
```diff
@@ -0,0 +1,15 @@
+
```
Comment (ret2libc) [inline aaa]: this should probably be removed, right?

Comment (ret2libc) [reply aaa]: i mean the entire file.

Comment (Obaied) [reply aaa]: I saw the file named `cmd_yank` so, I tried to keep the yanking functionality and just replace the removed `?y` with `yp`.

I can remove the file if its out of place, though

Comment (Obaied) [reply aaa]: Ah. I'm stupid. My eyes didn't notice this file I made when decoding b64. I just deleted it. Forget you saw anything :)

**Pull request merged**